Extension/test/scenarios/MultirootDeadlockTest/tests/quickInfo.test.ts

```diff
@@ -56,19 +64,19 @@ suite("[Quick info test]", function(): void {
         assert.strictEqual(actual, expected);
     });
 
-    // [TODO] - temporarily skip this test at the moment - it doesn't currently work (locally anyway) --
+    /** TODO: Investigate why this changes from <T> to int inconsistently */

```

Comment:
How is this skipped test failure being tracked? Should a GitHub issue be added with a link to it in the comment?

Reply:
I'll create issues to track these TODOs.